isis/src/mro/apps/marcical/marcical.cpp

```diff
@@ -508,7 +491,12 @@ namespace Isis {
           ocubeMgr[i] = icubeMgr[i];
         }
         else {
-          ocubeMgr[i] = stretch.Map(icubeMgr[i]) / (*fcubeMgrs[fcubeIndex])[i];
+          if ((*fcubeMgrs[fcubeIndex])[i] < 0.25) {
+            ocubeMgr[i] = 0;
+          }
+          else {
+            ocubeMgr[i] = stretch.Map(icubeMgr[i]) / (*fcubeMgrs[fcubeIndex])[i];
+          }
```

Comment:
This is the portion of the calibration logic missing from our implementation

See step 3 of https://pds-imaging.jpl.nasa.gov/data/mro/mars_reconnaissance_orbiter/marci/mrom_1343/calib/marcical.txt